src/model.ts

```diff
@@ -412,12 +410,12 @@ export namespace JudgeModel {
       this._metadata.set(key, value);
     }
 
+    version: string;
```

Comment:
It looks like version is declared as public. It is better to make it private, and expose a getter.